libbeat/docs/processors-config.asciidoc

```diff
@@ -371,3 +372,23 @@ _GCE_
   }
 }
 --------------------------------------------------------------------------------
+
+[[json-decode-fields]]
+===== json_decode_fields
+
+The `json_decode_fields` action decodes fields containing JSON strings and replaces the strings with valid JSON objects.
+
+[source,yaml]
+-----------------------------------------------------
+processors:
+ - json_decode_fields:
+     fields: ["field1", "field2", ...]
+     processArray: false 
+     maxDepth: 1
+-----------------------------------------------------
+
+The `json_decode_fields` action has the following configuration settings:
+
+`fields`:: The fields containing JSON strings to decode.
```

Comment:
@dedemorton Can you please mark this processor as experimental for 5.1 so that we can fix these config parameters in the next release. They should be named differently in the code to be consistent (like `process_array` instead of `processArray`).